backend/openai.js

```diff
@@ -99,6 +100,21 @@ async function chatGptChatCompletion() {
 }
 
 async function chatGptSendMessage(message) {
+  // keep track of any triggered defences
+  const defenceInfo = { blocked: false, triggeredDefences: [] };
+  // check if the message is too long
+  if (message.length > 280) {
```

Comment:
do we potentially want to have things like this configurable somewhere?

Reply:
Yeah that's a good idea! I'll make it an environment variable.

Reply:
Just pushed a fix for this.